### ISSUE TITLE
8x8 Matrix?

### ISSUE THREAD

**Issue by johnmastri** [MeULEDs/MeU_Square_GIF_Converter_Teensy#1]
Hey there - just a quick question before I dive in, it is as simple as changing the int values to 8 instead of 16 to generate the data for an 8x8 animate gif?

Thanks!

**Comment by MeULEDs** [comment]
Yes, change you can change the values on line 62 and line 63 to 8 and that should allow an 8x8 gif image to be converted.

If it doesn't work, let us know. Thanks!
